requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest==7.1.0
+coverage==6.3.2
+pre-commit==2.17.0
```

Comment:
I would add them in requirements-test.txt (are related to tests right?) except pre-commit